evolution/evaluation/evaluator.py

```diff
@@ -39,10 +43,8 @@ def __init__(self,
         self.context = context
         # Context Dataset outputs a scaled tensor and nonscaled tensor. The scaled tensor goes into PyTorch and
         # the nonscaled tensor is used to reconstruct the context that goes into enroads.
-        if set(context) == {"_global_population_in_2100",
-                            "_long_term_gdp_per_capita_rate",
-                            "_near_term_gdp_per_capita_rate",
-                            "_transition_time_to_reach_long_term_gdp_per_capita_rate"}:
+        # NOTE: These are the IDs for the context variables
+        if set(context) == {63, 235, 64, 236}:
```

Comment:
The comment is useful, but I still have no idea what `63` means or where to find what it means. A constant would help, as well as a link to where to find the info